test/realtime/database_test.exs

```diff
@@ -274,7 +274,7 @@ defmodule Realtime.DatabaseTest do
              end) =~ "project=123 external_id=123 [error] ErrorExecutingTransaction:"
     end
 
-    test "noproc error" do
+    test "noproc" do
```

Comment:
it would be better to something more descriptive

Reply:
Fixed in the right PR!